runtimes/spiritnet/Cargo.toml

```diff
@@ -67,6 +67,7 @@ pallet-transaction-payment = {git = "https://github.com/paritytech/substrate", d
 pallet-treasury = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
 pallet-utility = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
 pallet-vesting = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
+pallet-proxy = {branch = "polkadot-v0.9.13", default-features = false, git = "https://github.com/paritytech/substrate"}
```

Comment:
Same comments as in the other cargo file.

Reply:
done